_posts/samsr31/2017-05-22-sam-turtle-hack.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: samsr31
+title: "Turtle Hack Post - Sam Robinson"
+---
+
+Here is my Turtle Hack program:
+
+<iframe src="https://trinket.io/embed/python/6704f2abf5" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+When trying to think about what the “coolest” program I could write would be, I decided to mess around with the screen.onclick controls demonstrated in the first chapter of our video textbook.  I didn’t have any ideas for a specific drawing to try to create, and so instead I wanted to see if I could recreate the drawing functions from the textbook examples, and add some of my own.  Also, since I am not a particularly dextrous artist, I decided to try restricting tina’s movements in the drawing app, so that I would have more easy to work with angles.  At first, I had tina only turning in full 90 degree angles, but eventually decided to relax this to 45 degrees.  I feel like this gives a pretty good range of motion, while still being easy to judge tina’s position and exact heading.  Also, later on in the process I decided to add some penup and pendown options, as well as fill options.
```

Comment:
Nice! You'll be ahead of the game when we get to drawing programs and accepting user input.  The limitation of 45 degree turns is nice, and actually allows for some neat geometric designs.